.github/workflows/label.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler
+
+name: Label Community Request
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check for team membership
+      id: is_team_member
+      uses: jamessingleton/is-organization-member@v1
+      with:
+        organization: yugabyte
+        username: ${{ github.event.issue.user.login }}
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v2
```

Comment:
Why are we doing a checkout of repo?

Reply:
This is how all the examples for `actions-ecosystem/action-add-labels` are done, it seems as if its required.
https://github.com/marketplace/actions/actions-ecosystem-add-labels